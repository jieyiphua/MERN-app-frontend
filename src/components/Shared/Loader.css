#main-container {
  z-index: 1;
  position: absolute;
  margin-top: 0;
}
