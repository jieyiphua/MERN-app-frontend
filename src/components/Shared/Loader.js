import React from "react";
import "./Loader.css";

class Loader extends React.Component {
  render() {
    return (
      <div className="main-container" id="main-container">
        <div className="inner-main-container">
          <h1>Loading</h1>
        </div>
      </div>
    );
  }
}
export default Loader;
