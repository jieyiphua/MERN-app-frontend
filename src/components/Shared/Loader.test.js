import Loader from "./Loader";
import React from "react";
import Enzyme, { shallow, render, mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { createSerializer } from "enzyme-to-json";
import sinon from "sinon";

describe("<Loader />", () => {
  it("renders component", () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
