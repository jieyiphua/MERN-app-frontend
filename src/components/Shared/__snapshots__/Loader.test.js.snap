// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> renders component 1`] = `
<div
  className="main-container"
  id="main-container"
>
  <div
    id="loader-container"
  >
    <div
      class="spinner-border text-info"
      style={
        Object {
          "height": "100px",
          "width": "100px",
        }
      }
    />
    <div>
      <h1>
        B
      </h1>
      <h1>
        C
      </h1>
      <h1>
        M
      </h1>
      <h1>
        A
      </h1>
    </div>
  </div>
</div>
`;
